src/components/vuestic-components/va-icon/VaIcon.vue

```diff
@@ -46,6 +46,10 @@ const IconContextMixin = makeContextablePropsMixin({
     type: [String, Number],
     default: '',
   },
+  spin: {
```

Comment:
All props should be represented in demo.

Please also add all icontypes as well as svg/component icons to card to ensure spinning works properly in all cases.

Reply:
What is `icontypes`?

Reply:
Meant iconfont, sorry.

Reply:
Still not clear what `all cases` are meant? What `all cases`?